airbyte-cdk/bulk/toolkits/load-azure-blob-storage/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+    implementation project(':airbyte-cdk:bulk:core:bulk-cdk-core-base')
+    implementation project(':airbyte-cdk:bulk:core:bulk-cdk-core-load')
+    api project(':airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-aws')
+    api project(':airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-object-storage')
+
+    testFixturesApi(testFixtures(project(":airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-object-storage")))
+    implementation("aws.sdk.kotlin:s3:1.3.98")
+    implementation("aws.smithy.kotlin:http-client-engine-crt:1.3.31")
+    // https://mvnrepository.com/artifact/com.azure/azure-storage-blob
+    implementation("com.azure:azure-storage-blob:12.29.0")
+
+    testImplementation("io.mockk:mockk:1.13.16")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.11.3")
```

Comment:
Nit: consider making the version a constant just to make it easy to update.  See https://github.com/airbytehq/airbyte/blob/b0ae853f6476ae7fe46b828add6ef612dbabf6f8/airbyte-integrations/connectors/destination-mssql-v2/build.gradle.kts#L25 as an example.  5.11.4 is the latest.